packages/ethereum/web3-v4-ethereum/.npmignore

```diff
@@ -0,0 +1,2 @@
+src
```

Comment:
instead of npmignore files we should whitelist of files/folders for the package and you can do that in package.json "files" field (an array)